mllib-tests/v1p4/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -160,6 +166,18 @@ class GLMRegressionTest(sc: SparkContext) extends GLMTests(sc) {
         val rr = new RidgeRegressionWithSGD().setIntercept(addIntercept = true)
         rr.optimizer.setNumIterations(numIterations).setStepSize(stepSize).setRegParam(regParam)
         rr.run(rdd)
+      case ("l2", "elastic-net") =>
+        println("WARNING: Liner Regression with elastic-net in ML package ignores numIterations, " +
+          "stepSize in this Spark version.")
+        val rr = new LinearRegression().setElasticNetParam(elasticNetParam).setRegParam(regParam)
+        val sqlContext = new SQLContext(rdd.context)
+        import sqlContext.implicits._
+        val mlModel = rr.fit(rdd.toDF())
```

Comment:
Note: generating the DF inside of `runTests` is OK for now, but it introduces the DF generation overhead into the performance metrics for ML pipeline algorithms. We will eventually need to refactor spark-perf to use DataFrames.